avro-codegen/src/test/java/com/linkedin/avroutil1/codegen/SpecificRecordClassGeneratorTest.java

```diff
@@ -31,4 +31,18 @@ public void testSimpleEnum() throws Exception {
 
     CompilerHelper.assertCompiles(javaSourceFile);
   }
+
+  @Test
+  public void testHugeEnum() throws Exception {
```

Comment:
Is there any way we can test against the generated code? Is that desirable?

Reply:
there is, and it is desirable. at the end of this i assert that the code compiles, but you cold load the resulting class and act on it using reflection